src/server/system_services/bucket_server.js

```diff
@@ -1323,32 +1321,9 @@ async function get_cloud_buckets(req) {
 
 
 async function update_all_buckets_default_pool(req) {
-    const pool_name = req.rpc_params.pool_name;
-    const pool = req.system.pools_by_name[pool_name];
-    if (!pool) throw new RpcError('INVALID_POOL_NAME');
-    const internal_pool = pool_server.get_internal_mongo_pool(pool.system);
-    if (!internal_pool || !internal_pool._id) return;
-    if (String(pool._id) === String(internal_pool._id)) return;
-    const buckets_with_internal_pool = _.filter(req.system.buckets_by_name, bucket =>
-        is_using_internal_storage(bucket, internal_pool));
-    if (!buckets_with_internal_pool.length) return;
-
-    const updates = [];
-    for (const bucket of buckets_with_internal_pool) {
-        updates.push({
-            _id: bucket.tiering.tiers[0].tier._id,
-            mirrors: [{
-                _id: system_store.new_system_store_id(),
-                spread_pools: [pool._id]
-            }]
-        });
-    }
-    dbg.log0(`Updating ${buckets_with_internal_pool.length} buckets to use ${pool_name} as default resource`);
-    await system_store.make_changes({
-        update: {
-            tiers: updates
-        }
-    });
+    dbg.warn('update_all_buckets_default_pool is deprecated and will be removed in the next release');
```

Comment:
Can we open a gap here? So we will remember to remove this code also from the operator side.